infra/coreml_export/process_results.py

```
@@ -0,0 +1,123 @@
+    dynamo = DynamoClient(table_name=table_name)
+    records = event.get("Records", [])
+
+    processed = 0
+    errors: List[str] = []
+
+    for record in records:
+        try:
+            message = json.loads(record["body"])
+            process_export_result(dynamo, message)
+            processed += 1
+        except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)